add _posts/2021-10-04-sharing-materials-from-ncwit-talk.md (#506)
USRSE/usrse.github.io: _posts/2021-10-04-sharing-materials-from-ncwit-talk.md

---
layout: post

> title: Sharing resources on counteracting bias from recent NCWIT talk

Reviewer: ```suggestion
title: Sharing resources on counteracting bias, from recent NCWIT talk
```

Author: good catch, we don't it to sound like we are counteracting bias from the talk 😬 

Re-ordered the title: "Sharing resources on counteracting bias from recent NCWIT talk"

tags: [events, DEI, working groups]
posted_by: David Nicholson
---

On Thursday, August 12th, US-RSE held the
[second talk of our speaker series](https://us-rse.org/events/2021/2021-07-dei-speaker-series-ncwit)
organized by the Diversity, Equity, and Inclusion (DEI) working group.

> We were joined by [Joanne Esch](https://us-rse.org/events/2021/2021-07-dei-speaker-series-ncwit)

Reviewer 2: This seems like a good link: https://ncwit.org/profile/joanne-esch/.

> We were joined by [Joanne Esch](https://us-rse.org/events/2021/2021-07-dei-speaker-series-ncwit)

Author: whoops, that's actually the link I wanted there, thank you -- Ctrl-C/V error

and [Andrea Bowen-Jones](https://ncwit.org/profile/andrea-bowens-jones/)
from [The National Center for Women & Information Technology (NCWIT)](https://ncwit.org/).

> They spoke with us the biases and barriers that

Reviewer: ```suggestion
They spoke with us about the biases and barriers that
```
(or similar)

Author: Thank you! I managed to introduce some new "bugs" revising :(
Fixed

aimed at addressing the biases and barriers that
face women and other underrepresented groups
participating in the technical workforce.

We would like to pass along some of the resources that the speakers shared.
The first resource is the NCWIT's
[critical listening guide](https://ncwit.org/resources/critical-listening-guide/).
This guide addresses common misunderstandings that arise during discussions of

> the participation of women and other underrepresent group in technology.

Reviewer: ```suggestion
the participation of women and other underrepresented groups in technology.
```

Author: Thank you!

Author: I removed this phrase since I used it in the sentence above.

In the guide you'll find concrete responses and links to primary research
to help you deal with problematic statements, like "Women need to learn to be

> more confident". The guide provides questions to ask and comments to add

Reviewer 3: Periods, commas, etc. usually go on the inside of quotes.
```suggestion
more confident." The guide provides questions to ask and comments to add
```

Author: You are right, thank you

more confident." The guide provides questions to ask and comments to add
that can prevent these problematic statements from derailing a discussion.

> Two more guides we'd like to point are aimed specifically at interrupting bias

Reviewer: ```suggestion
Two more guides we'd like to highlight are aimed specifically at interrupting bias
```
or point out or whatever

Reviewer: "interrupting bias" is an interesting verb for it but I surmise that it has currency in the community. Naively I'd have imagined that it's the instances of bias that are to be interrupted. Still, not saying to change anything, just thinking out loud.

Author: "highlight" is a better word than "point out" anyway, thank you

[in industry](https://ncwit.org/resource/biasindustry/)
and [in academic settings](https://ncwit.org/resource/biasacademic/).

The DEI working group is building a collection of resources like these
so that we can learn from organizations like NCWIT
and help US-RSE uphold the values stated
in our [DEI mission statement](https://us-rse.org/dei-mission/).
These resources are guiding our own discussions,

> along with other material like [books we've read in our media discussion group]

Reviewer 3: ```suggestion
along with other material like [books we've read in our media discussion group](https://www.eventbrite.com/x/better-allies-book-discussion-part-i-tickets-151687323869?utm_campaign=order_confirm&utm_medium=email&ref=eemailordconf&app_cta_src=order_conf_email&utm_source=eventbrite&utm_term=digitalx).
```

> (https://www.eventbrite.com/x/better-allies-book-discussion-part-i-tickets-151687323869?utm_campaign=order_confirm&utm_medium=email&ref=eemailordconf&app_cta_src=order_conf_email&utm_source=eventbrite&utm_term=digitalx).

Reviewer 3: ```suggestion
```
